push/service.go

```diff
@@ -55,17 +89,34 @@ func newClient(cert tls.Certificate) (*http.Client, error) {
 	return &http.Client{Transport: transport}, nil
 }
 
-// Push notification to APN service after performing serialization.
-func (s *Service) Push(deviceToken string, headers *Headers, payload interface{}) (string, error) {
+// Push notification queues a notification APN service after performing serialization.
```

Comment:
Because this could return an error before queuing the notification, I'm thinking of removing this version and just have people use `json.Marshal` (and rename PushBytes to Push).